tests/tests_suite_test.go

```diff
@@ -24,16 +22,3 @@ func TestTests(t *testing.T) {
 	RunSpecsWithDefaultAndCustomReporters(t, "Tests Suite", reporters)
 }
 
-var _ = BeforeSuite(func() {
```

Comment:
I think we should leave these, even though Framework should clean up all the namespaces, hard kills or other issues outside of the Frameworks control could leave namespaces hanging. This code is simply a backup to clean up anything that didn't get cleaned up for whatever reason. In most cases it won't do anything and thus there is no harm in having it.

Reply:
@jeffvance This is the only thing left I think we should put back, basically a backup cleaner in case the primary one from the framework failed for whatever reason.